src/scrot.c

```diff
@@ -131,16 +130,24 @@ int main(int argc, char *argv[])
         else
             image = scrotGrabShot();
     }
-
     if (!image)
         errx(EXIT_FAILURE, "no image grabbed");
 
+    /* Get the time right after the screenshot.
+     * Don't put any new code between this clock_gettime() call and the
+     * screenshot-taking above or it will skew the timing.
+     */
+    clock_gettime(CLOCK_REALTIME, &timeStamp);
+    if (timeStamp.tv_nsec >= 500*1000*1000) {
+        /* Round the timestamp to the nearest second, this reduces the maximum
+         * timing error from discarding tv_nsec from 999ms to 500ms.
+         */
```

Comment:
Just "Round the timestamp to the nearest second" is fine. The rest should be fairly obvious and don't require a comment.